test/Spec/State/Computed.spec.luau

```diff
@@ -57,7 +57,7 @@ return function()
 		
 		local scope = {}
 		local dependency = Value(scope, 5 :: number | string)
-		local computed = Computed(scope, function(use)
+		local computed = Computed(scope, function(use, scope)
 			return use(dependency)
```

Comment:
Scope shouldn't be provided as it is never used
```suggestion
		local computed = Computed(scope, function(use)
```

Reply:
reverted